src/scripts/gym/Champion.ts

```diff
@@ -18,22 +18,22 @@ class Champion extends Gym {
         super(leaderName, town, basePokemon, badgeReward, moneyReward, rewardMessage, requirements, rewardFunction);
     }
 
-    public setPokemon(starter: GameConstants.Starter) {
+    public setPokemon(starter: number) {
         this.pokemons = [...this.basePokemon];
         switch (starter) {
-            case GameConstants.Starter.Bulbasaur: {
```

Comment:
Are these still used anywhere now? or should they be removed from GameConstants too?